src/Cake.Core/Scripting/ScriptConventions.cs

```diff
@@ -20,18 +20,21 @@ public sealed class ScriptConventions : IScriptConventions
         private readonly IFileSystem _fileSystem;
         private readonly IAssemblyLoader _loader;
         private readonly ICakeRuntime _runtime;
+        private readonly IReferenceAssemblyResolver _referenceAssemblyResolver;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ScriptConventions"/> class.
         /// </summary>
         /// <param name="fileSystem">The file system.</param>
         /// <param name="loader">The assembly loader.</param>
         /// <param name="runtime">The Cake runtime.</param>
-        public ScriptConventions(IFileSystem fileSystem, IAssemblyLoader loader, ICakeRuntime runtime)
+        /// <param name="referenceAssemblyResolver">The reference assembly resolver.</param>
+        public ScriptConventions(IFileSystem fileSystem, IAssemblyLoader loader, ICakeRuntime runtime, IReferenceAssemblyResolver referenceAssemblyResolver)
```

Comment:
@devlead does this constitute a breaking change?

Reply:
Yes, and we've bumped the minimal Cake version already.

Though think very few use this part of Cake externally, as it's compiling part of scripts.

Reply:
Ok, makes sense. I will update the linked issue with the breaking change label, just so it is clear.